src/components/Table/components/TH/TH.style.ts

```diff
@@ -0,0 +1,24 @@
+import type { SerializedStyles, Theme } from '@emotion/react';
+import { css } from '@emotion/react';
+
+import { getMinHeight } from '../TD/TD.style';
+import type { TableProps } from 'components/Table';
+import { generateStylesFromTokens } from 'components/Typography/utils';
+
+/** @TODO replace all css with tokens */
+
+export const thContainer =
+  ({ rowSize, width }: Pick<TableProps<any>, 'rowSize'> & { width?: number }) =>
+  (theme: Theme): SerializedStyles => {
+    return css`
+      width: ${width ? `${width}%` : undefined};
+      height: ${getMinHeight(rowSize)(theme)};
+      align-content: center;
+      box-sizing: border-box;
+      padding: 8px 16px;
```

Comment:
same here? no token for that?

Reply:
tokens will be added after #797 and the rest of the parts are merged